__mocks__/react-redux.js

```diff
@@ -0,0 +1,5 @@
+export const useDispatch = jest.fn();
+
+export const useSelector = jest.fn((selector) => selector({
+  tasks: [],
+}));
```

Comment:
```suggestion
export const useSelector = jest.fn();
```

이렇게만 해도 충분할 거에요. 실제로 mockImplementation을 각 테스트에서 구현하고 있으니까요.